front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/Areas/EmptyState.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { Box, Text, useBreakpoint } from '@citizenlab/cl2-component-library';
+
+import useAreaTerms from 'hooks/areaTerms/useAreaTerms';
+
+import { DEFAULT_PADDING } from 'components/admin/ContentBuilder/constants';
+
+import { useIntl } from 'utils/cl-intl';
+
+import messages from './messages';
+
+const EmptyState = () => {
+  const isSmallerThanPhone = useBreakpoint('phone');
+  const { areasTerm } = useAreaTerms();
+
+  const { formatMessage } = useIntl();
+
+  return (
+    <Box px={isSmallerThanPhone ? DEFAULT_PADDING : undefined}>
+      <Text color="textSecondary">
+        {formatMessage(messages.thereAreCurrentlyNoProjects, {
```

Comment:
Not a huge issue, but I think there are two empty states that we should distinguish:
1. You are not seeing any projects because there are no projects in the areas that you follow
2. You are not seeing an projects because you are not currently following any areas

I think this empty state message is fine for number 1, but for number 2 we could maybe have something like a call to action? E.g. "You are not seeing any projects because you are not following any areas. Click *here* to follow some areas" (with `here` opening the modal)

Reply:
> I think this empty state message is fine for number 1, but for number 2 we could maybe have something like a call to action? 

- Yeah, we could make a second empty state. The second empty state you're suggesting is however ~already captured by the first. If you follow no areas, there are no projects in the areas that you follow. But I get it's not entirely the same. 
- There could certainly be a CTA within the empty state. That's [what I've asked in this ticket](https://www.notion.so/govocal/Empty-state-1bc9663b7b26802dace5d227d55f95ea?pvs=4).
- We currently also show archived projects while I speak of "active projects" in the empty state.

So yeah, this can be improved. I'm adding this to that ticket. There are a few other loose ends and I'll make a summary for product.

Reply:
@luucvanderzee